docs/victory-candlestick/ecology.md

```diff
@@ -0,0 +1,240 @@
+VictoryCandlestick
+=============
+
+Draw SVG bar charts with [React][]. VictoryCandlestick is a composable component, so it doesn't include axes. Check out [VictoryChart][] for complete candlestick charts and more.
+
+## Features
+
+### Props are Optional
+
+VictoryCandlestick is written to be highly configurable, but it also includes a set of sensible defaults and fallbacks. If no props are provided, VictoryCandlestick will plot sample data.
+
+``` playground
+<VictoryCandlestick/>
+```
+
+To display your own data, just pass in an array of data objects, or an array of arrays via the data prop.
+
+```playground
+<VictoryCandlestick
+  data={[
+    {x: new Date(2016, 6, 1), open: 5, close: 10, high: 15, low: 0},
+    {x: new Date(2016, 6, 2), open: 15, close: 10, high: 20, low: 5},
+    {x: new Date(2016, 6, 3), open: 15, close: 20, high: 25, low: 10},
+    {x: new Date(2016, 6, 4), open: 20, close: 25, high: 30, low: 15},
+    {x: new Date(2016, 6, 5), open: 30, close: 25, high: 35, low: 20}
+  ]}
+/>
+```
+
+VictoryCandlestick comes with data accessor props to make passing in data much more flexible.
+assign a property to x, open, close, high, or low, or process data on the fly.
+
+```playground
+<VictoryCandlestick
+  data={[
+    {date: new Date(2016, 8, 2), open: 5, close: 10, low: 0},
+    {date: new Date(2016, 8, 3), open: 15, close: 10, low: 5},
+    {date: new Date(2016, 8, 4), open: 15, close: 20, low: 10},
+    {date: new Date(2016, 8, 5), open: 25, close: 20, low: 15},
+    {date: new Date(2016, 8, 6), open: 25, close: 30, low: 20}
+  ]}
+  x={"date"}
+  high={(data) => (Math.max(data.open, data.close) + 10)}
+/>
+```
+
+### Flexible and Configurable
+
+The sensible defaults VictoryBar provides makes it easy to get started, but everything can be overridden, and configured to suit your needs:
+
+```playground
+<VictoryChart
+  height={600}
+  padding={75}
+  style={{
```

Comment:
and an explicit domain, or domain padding so that the candlesticks dont render over the axis